conan/tools/cmake/utils.py

```diff
@@ -2,3 +2,12 @@ def is_multi_configuration(generator):
     if not generator:
         return False
     return "Visual" in generator or "Xcode" in generator or "Multi-Config" in generator
+
+
+def get_file_name(conanfile):
+    """Get the name of the file for the find_package(XXX)"""
+    ret = conanfile.new_cpp_info.get_property("cmake_file_name", "CMakeDeps")
```

Comment:
This is not very great now, that it depends explicitly on ``CMakeDeps``, while it has been made common and is being used not in CMakeDeps, but in CMakeToolchain. I understand why, but reads a bit confusing if you don't know it.

Reply:
It is used in both of them.

Reply:
I added a comment. Suggestion welcome.

Reply:
I am not fully convinced on the proposition of both having the property name and the specific generator as a specialization. We might want to revisit that, but yes, I guess at the moment this is fine here.